packages/shared/src/dto/workflows/step.dto.ts

```diff
@@ -84,6 +85,7 @@ export enum UiComponentEnum {
   CHAT_BODY = 'CHAT_BODY',
   PUSH_BODY = 'PUSH_BODY',
   PUSH_SUBJECT = 'PUSH_SUBJECT',
+  SKIP_RULES = 'SKIP_RULES',
```

Comment:
was not sure how to call. should we switch to SKIP?

```suggestion
  SKIP = 'SKIP',
```

Reply:
```suggestion
  SKIP_RULES = 'QUERY_EDITOR',
```